README.md

```diff
@@ -13,7 +13,13 @@ Requires a configuration file `configuration.json` with information for each eng
         "port" : 7000,
         "processes" : 8,
         "enabled" : true,
-        "verbose": true
+        "verbose": true,
+        "testTimeout": 60,
+        "testCount" : 100,        
+        "testSorting": "Fair",
+        "queryComplexity": [5, 10],
+        "omitTests": true,
+        "testsSavePath": "/path/to/file"
```

Comment:
Maybe we should consider populating the attributes with their default values? E.g. "testTimeout": 30?